Tools-Override/FrameworkTargeting.targets

```diff
@@ -228,11 +228,18 @@
 
   <Target Name="BinPlaceFiles"
           Condition="'@(BinPlaceDir)' != ''"
-          DependsOnTargets="GetBinPlaceItems" >
+          DependsOnTargets="GetBinPlaceItems"
+          Inputs="@(BinPlaceDir);%(BinPlaceDir.ItemName)"
```

Comment:
As far as I remember Inputs do nothing without actual Outputs. If you just want to batch, the convention is to put it on the Condition which would also eliminate your existing condition

` Condition="'%(BinPlaceDir.ItemName)' != ''"`